themes/ddbasic/sass/components/form/form-user-profile.scss

```diff
@@ -91,7 +91,7 @@
         clear: left;
       }
 
-      #edit-pass--2 {
+      fieldset[id^='edit-pass'] {
```

Comment:
Isn't it possible to just have `[id^='edit-pass']` without the fieldset?

Reply:
True. Personally, though, I like knowing what kind of element we're looking for...

Reply:
Good point :)